helm/alfresco-process-application/values.yaml

```diff
@@ -382,3 +382,10 @@ rabbitmq:
       nginx.ingress.kubernetes.io/enable-cors: "true"
       nginx.ingress.kubernetes.io/cors-allow-headers: "*"
       nginx.ingress.kubernetes.io/x-forwarded-prefix: "true"
+
+persistence:
+  enabled: false
+  storageClassName: default-sc
```

Comment:
this matches with the storage class name created by @mmartinadanx in the installation right?

Reply:
yes that's the default name we provide during the installation of Infrastructure chart